pom.xml

```diff
@@ -1220,6 +1223,11 @@
                 <module>gradle-modules/gradle/maven-to-gradle</module>
                 <module>persistence-modules/spring-data-neo4j</module>
                 <module>spring-actuator</module>
```

Comment:
Please remove spring-cloud-contract. 
This will be added by another ongoing jira.

Reply:
done